volatility3/framework/plugins/linux/lsof.py

```diff
@@ -145,6 +151,7 @@ def list_fds(
         context: interfaces.context.ContextInterface,
         vmlinux_module_name: str,
         filter_func: Callable[[int], bool] = lambda _: False,
+        include_files_only: bool = False,
```

Comment:
This is a change of a class method, but in a backwards compatible way, so it still needs the plugin MINOR version bumping (ie, `(2, 1, 0)`)